src/DynamoCore/Properties/Resources.resx

```diff
@@ -735,4 +735,10 @@ Parameter name: {0}</value>
     <value>The current user, '{0}', is not a maintainer of the package '{1}'.</value>
     <comment>{0} = user name (i.e. 'DynamoTeam'), {1} = package name (i.e. 'Clockwork for Dynamo 1.x')</comment>
   </data>
+  <data name="DuplicatedOlderPackage" xml:space="preserve">
+    <value>An older version of the package called {0} version {2} was found at {1} with version {3}. Ignoring it.</value>
+  </data>
+  <data name="InvalidPackageVersion" xml:space="preserve">
+    <value>The version of the package called {0} found at {1} is invalid (version: "{2}"). Ignoring it.</value>
+  </data>
```

Comment:
both Resources.en-US.resx  and Resources.resx are needed ?

Reply:
As far as I know, yes. @mjkkirschner?

Reply:
yes